### PR TITLE
Add bridge-marker bump script

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
Currently cnao only bumps bridge-marker manually.
    Let's add a script to do it for us.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by RamLavi** [comment]
/hold

until #449 is merged

File: data/linux-bridge/003-bridge-marker-rbac.yaml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-apiVersion: extensions/v1beta1
```
Comment (ormergi) [inline data/linux-bridge/003-bridge-marker-rbac.yaml:1]: Is that not necessary anymore?

Comment (RamLavi) [reply data/linux-bridge/003-bridge-marker-rbac.yaml:1]: it's a duplication, see 2 lines below there is already `apiVersion: rbac.authorization.k8s.io/v1beta1`

**Comment by RamLavi** [comment]
/unhold

added manifest creation

File: hack/components/bump-bridge-marker.sh [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+rm -f data/linux-bridge/0004*.yaml #remove old file
+cp ${BRIDGE_MARKER_PATH}/config/cnao/003-bridge-marker.yaml data/linux-bridge/
+
+echo 'Get bridge-marker image name and update it under CNAO'
+BRIDGE_MARKER_TAG=$(git-utils::get_component_tag ${BRIDGE_MARKER_PATH})
+BRIDGE_MARKER_IMAGE=quay.io/kubevirt/bridge-marker
+BRIDGE_MARKER_IMAGE_TAGGED=${BRIDGE_MARKER_IMAGE}:${BRIDGE_MARKER_TAG}
+sed -i "s#\"${BRIDGE_MARKER_IMAGE}:.*\"#\"${BRIDGE_MARKER_IMAGE_TAGGED}\"#" \
+    pkg/components/components.go \
+    test/releases/${CNAO_VERSION}.go
```
Comment (qinqon) [inline hack/components/bump-bridge-marker.sh:97]: Missing new line

File: hack/components/bump-bridge-marker.sh [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+seLinuxContext:
+  type: RunAsAny
+users:
+- system:serviceaccount:{{ .Namespace }}:bridge-marker
+{{ end }}
+---
+EOF
+
+	echo 'rejoin sub-manifests to final manifest'
+	YAML_FILE=003-bridge-marker.yaml
```
Comment (qinqon) [inline hack/components/bump-bridge-marker.sh:77]: Why 003 we have only one manifests

File: data/linux-bridge/003-bridge-marker.yaml [hunk 4]
```diff
@@ -0,0 +1,89 @@
+---
```
Comment (qinqon) [inline data/linux-bridge/003-bridge-marker.yaml:1]: Where is Namespace ?

File: hack/components/bump-bridge-marker.sh (hunk 3 above)
Comment (RamLavi) [reply hack/components/bump-bridge-marker.sh:77]: it is kept on the linux bridge folder:
```
cluster-network-addons-operator (add_bridge_marker_script) $ ls data/linux-bridge/
000-ns.yaml  001-rbac.yaml  002-linux-bridge.yaml  003-bridge-marker.yaml
```

File: data/linux-bridge/003-bridge-marker.yaml (hunk 4 above)
Comment (RamLavi) [reply data/linux-bridge/003-bridge-marker.yaml:1]: Do you mean [this](https://github.com/kubevirt/cluster-network-addons-operator/pull/458/files/61c4cb119f80b2890336cf0ab5933eb275ac00b4#diff-0570096c4288aecf7e8ceb569e77cf7eR6)?

Comment (RamLavi) [reply data/linux-bridge/003-bridge-marker.yaml:1]: i think we have all the Namespaces

**Pull request merged**